### PR TITLE
update trusted icons and make trusted icon appear on the editor title

### PR DESCRIPTION
**Pull request by DavidKutu**
For #4338, #4339

Here's what the new trusted icon looks like (tooltip says 'Trusted' and clicking does nothing):
image

And here's the new untrusted icon (brings up the trust notification):
image

-   [ ] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [x] Title summarizes what is changing.
-   [x] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [ ] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [ ] Has telemetry for enhancements.
-   [ ] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

**Review by rchiodo — APPROVED** [review]

**Review by IanMatthewHuff — CHANGES_REQUESTED** [review]
Code is fine just wanted to discuss quick that I think this is going to be an accessibility violation. This command is going to show up as a clickable button to a screen reader. And it's going to have no effect when run. Are we ok with that?

**Comment by DavidKutu** [comment]
Claudia's ruling: have the notification pop on saying its already trusted.
"This notebook is already trusted."

**Review by IanMatthewHuff — APPROVED** [review]

File: src/client/datascience/interactive-ipynb/trustCommandHandler.ts [hunk 1]
```diff
@@ -41,6 +41,13 @@ export class TrustCommandHandler implements IExtensionSingleActivationService {
         const context = new ContextKey('jupyter.trustfeatureenabled', this.commandManager);
         context.set(true).ignoreErrors();
         this.disposables.push(this.commandManager.registerCommand(Commands.TrustNotebook, this.onTrustNotebook, this));
+        this.disposables.push(
```
Comment (DonJayamanne) [inline src/client/datascience/interactive-ipynb/trustCommandHandler.ts:44]: I think we should get this reviewed by a PM.
Generally I ton'd think its a good idea to display such messages. One could argue if the title indicates its trusted, then wy allow clickcing the button just to always (100% of time) display a message.
If the message is required, then i'd argue the meaning of the icon is not obvious to users.

@claudiaregio /cc

**Review by DonJayamanne — CHANGES_REQUESTED** [review]
Would like to get @claudiaregio input on this.
I do'nt thinnk the button should be clickable to display messages that do absolutely nothing.

File: src/client/datascience/interactive-ipynb/trustCommandHandler.ts (hunk 1 above)
Comment (IanMatthewHuff) [reply src/client/datascience/interactive-ipynb/trustCommandHandler.ts:44]: @DonJayamanne David talked to Claudia per his comment above.

Comment (DavidKutu) [reply src/client/datascience/interactive-ipynb/trustCommandHandler.ts:44]: > Claudia's ruling: have the notification pop on saying its already trusted.
"This notebook is already trusted."

**Review by DonJayamanne — APPROVED** [review]

**Review by DonJayamanne — CHANGES_REQUESTED** [review]
Spoke to claudia and we need an icon that cannot be clicked.
I.e. just disable it and display tooltip.
She just spoke to me and she was under the impression we cannot disable it, which we can.

File: src/client/datascience/interactive-ipynb/trustCommandHandler.ts (hunk 1 above)
Comment (DonJayamanne) [reply src/client/datascience/interactive-ipynb/trustCommandHandler.ts:44]: Claudia just pinged me & stated that we need an icon that cannot be clicked.
I.e. just disable it and display tooltip.

She was under the impression we cannot disable it

**Review by DonJayamanne — APPROVED** [review]

**Pull request merged**